gradle.properties

```diff
@@ -2,22 +2,20 @@ KOTLIN_VERSION=1.4.20
 BINTRAY_VERSION=1.8.3
 ARTIFACTORY_VERSION=4.17.2
 KTLINT_VERSION=9.2.1
-
 JACOCO_VERSION=0.8.3
-
 KOTLIN_COROUTINES_VERSION=1.4.2
 SL4J_VERSION=1.7.25
 JODA_VERSION=2.9.7
 JODA_CONVERT_VERSION=1.8.1
 NETTY_VERSION=4.1.49.Final
 KOTLIN_LOGGING_VERSION=1.7.6
+SCRAM_CLIENT_VERSION=1.0.0-beta.2
 R2DBC_SPI_VERSION=0.8.0.RELEASE
 REACTOR_CORE_VERSION=3.2.6.RELEASE
-
 JUNIT_VERSION=4.12
 ASSERTJ_VERSION=3.11.1
 MOCKK_VERSION=1.9
 LOGBACK_VERSION=1.1.8
-TEST_CONTAINERS_VERSION=1.15.0
+TEST_CONTAINERS_VERSION=1.15.1
```

Comment:
This new version [fixes](https://github.com/testcontainers/testcontainers-java/issues/3574) a Windows compatibility issue.